Kickstarter-iOS/Features/ProjectPage/Controller/ProjectPageViewControllerTests.swift

```diff
@@ -41,12 +41,12 @@ internal final class ProjectPageViewControllerTests: TestCase {
           caption: "camera"
         ),
         AudioVideoViewElement(
-          sourceURLString: "https://source.com",
+          sourceURLString: "https://not-a-real-url-dot-com",
```

Comment:
This ensures that the test doesn't actually load any content here, so the results will always be the same. Note that `https://source.com` isn't a video to begin with, which is why it shows a blank video player.